src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -93,6 +93,7 @@ object ProtocPlugin extends AutoPlugin {
 
     PB.includePaths := PB.includePaths.?.value.getOrElse(Nil),
     PB.includePaths ++= PB.protoSources.value,
+    PB.includePaths ++= protocIncludeDependencies.value,
```

Comment:
Let's add a way for users to modify this list so they can assign Nil to it in case this causes problems. Can you make this `Seq` a setting under the `PB` object? Let's call it `PB.dependentProjectsIncludePath`